guides/source/active_storage_overview.md

```diff
@@ -721,10 +720,11 @@ Analyzing Files
 
 Active Storage analyzes files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
 
-Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, and `display_aspect_ratio`.
-Audio analysis provides `duration` and `bit_rate` attributes.
+Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, `display_aspect_ratio` and presence/absence of `video` and `audio` channels. Audio analysis provides `duration` and `bit_rate` attributes.
```

Comment:
```suggestion
Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, `display_aspect_ratio`, and `video` and `audio` booleans to indicate the presence of those channels. Audio analysis provides `duration` and `bit_rate` attributes.
```

Reply:
Ooooops. Sorry about missing that. It's fixed now.